lib/ansible/modules/net_tools/nios/nios_host_record.py

```diff
@@ -178,6 +199,7 @@ def main():
 
     ib_spec = dict(
         name=dict(required=True, ib_req=True),
+        old_name=dict(required=False, ib_req=True),
```

Comment:
Instead of having a separate option for `old_name` for sake of better UI can we modify `name` to accept either a string type that is the `name` entry to be created or a `dict` that accepts old name and the new name in case if name entry needs to be updated. The `type` in this case will be a callable and need to be documented for accepted inputs.

Thoughts/Suggestions?